map/bookmark_manager.cpp

```diff
@@ -2326,6 +2340,15 @@ kml::MarkGroupId BookmarkManager::CreateBookmarkCategory(std::string const & nam
   return groupId;
 }
 
+void BookmarkManager::ReloadBookmarkCategory(kml::MarkGroupId & groupId, kml::CategoryData && data, bool autoSave /* = true */)
+{
+  CHECK_THREAD_CHECKER(m_threadChecker, ());
+  CHECK_NOT_EQUAL(m_categories.count(groupId), 0, ());
+  ClearGroup(groupId);
+  m_categories.emplace(groupId, std::make_unique<BookmarkCategory>(std::move(data), autoSave));
+  m_changesTracker.OnAddGroup(groupId);
+}
+
 BookmarkCategory * BookmarkManager::CreateBookmarkCompilation(kml::CategoryData && data)
```

Comment:
Is Compilation term different from Category? Can compilation contain many Cagegories? Does it make sense to refactor/rename it separately to avoid confusion?

Reply:
Compilations == Categories. They have the same type:
`  CategoriesCollection m_compilations;`

But I didn't think that we should rename it because there are a lot of the code related to this term.